lua/mappings.lua

```diff
@@ -24,6 +24,45 @@ map("v", "x", [=[ "_x ]=], opt)
  this line too ]]
 --
 
+-- toggle unix readline's keymap
+map("n", user_map.unix_keymap.toggle_unix_keymap,"<cmd>lua require 'mappings'.unix_keymap()<CR>",{nowait = true})
+
+local _cmap_containp = function (key)
+  local cmap_tab = vim.api.nvim_get_keymap("c")
+
+  for _, value in ipairs(cmap_tab) do
+    if value['lhs'] == key then
+      return true
+    end
+  end
+  return false
+end
+
+M.unix_keymap = function()
+  local m = user_map.unix_keymap
+
+  if _cmap_containp("<C-A>") then
```

Comment:
Yes to if, but change the conditional to checking if the option in `chadrc.lua` `options`